### PR TITLE
Clean up directory structure

### LINKED ISSUE #25: Clean up directories
- `libs` -> `src` (as is NPM convention)
- remove `tmp` and `dist`

### PR DESCRIPTION
**Pull request by hobofan**
Fixes #25

### CODE REVIEW

File: src/Contracts.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const Web3 = require('web3');
-const AbiBinProvider = require('../libs/AbiBinProvider');
+const AbiBinProvider = require('../src/AbiBinProvider');
```
Comment (schemar) [inline src/Contracts.js]:
```suggestion
const AbiBinProvider = require('./AbiBinProvider');
```
🤔

File: src/helpers/StakeHelper.js [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
 const Web3 = require('web3');
 const BN = require('bn.js');
-const Contracts = require('../../libs/Contracts');
+const Contracts = require('../../src/Contracts');
```
Comment (schemar) [inline src/helpers/StakeHelper.js]:
```suggestion
const Contracts = require('../Contracts');
```
🤔

File: src/utils/EIP712SignerExtension/extender.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ let extend = function() {
   }
 
   const Account = require('eth-lib/lib/account');
-  const TypedData = require('../../../libs/utils/EIP712SignerExtension/TypedData');
+  const TypedData = require('../../../src/utils/EIP712SignerExtension/TypedData');
```
Comment (schemar) [inline src/utils/EIP712SignerExtension/extender.js]:
```suggestion
  const TypedData = require('../../utils/EIP712SignerExtension/TypedData');
```
🤔

**Review by schemar — APPROVED** [review]
Nice 👍 
Then where would we put the cli?

No updates to `package.json` scripts required?
Some imports are unnecessarily complex (see comments).

**Comment by hobofan** [comment]
> Then where would we put the cli?

That would go into `bin`, with the bulk of the cli code living under `src/cli`.

> No updates to package.json scripts required?

Nope, `main` already pointed to `index.js` (which makes me think that `dist` wasn't even really in use anyway).

**Pull request merged**